spec/models/test_track/web_session_spec.rb

```diff
@@ -19,7 +19,8 @@ def current_clown; end
 
   let(:cookies) { { tt_visitor_id: "fake_visitor_id" }.with_indifferent_access }
   let(:headers) { {} }
-  let(:request) { double(:request, host: "www.foo.com", ssl?: true, headers: headers) }
+  let(:request) { double(:request, host: "www.foo.com", ssl?: true, headers: headers, local?: local_request?) }
+  let(:local_request?) { false }
```

Comment:
this naming is a little confusing though, given the `ApplicationController#local_request?` method which loops in the consider_all_requests_local config - maybe just call this `local?`

Reply:
You're right. I was trying to make it clear that this method was for the request only, but it introduces other ambiguity. Renaming it to `local?`.